app/src/main/res/values-in/strings.xml

```diff
@@ -12,34 +12,36 @@
     <string name="safetynet">SafetyNet</string>
 
     <!--Home-->
-    <string name="no_connection">Koneksi tidak tersedia</string>
+    <string name="no_connection">Tidak tersedia koneksi internet</string>
     <string name="app_changelog">Catatan perubahan</string>
     <string name="loading">Memuat…</string>
     <string name="update">Update</string>
     <string name="not_available">N/A</string>
     <string name="hide">Tutup</string>
-    <string name="home_package">Paket</string>
+    <string name="home_package">ID Paket</string>
+    <string name="home_app_title">Aplikasi</string>
```

Comment:
Original string in English is "App", so can we take the shorter form of "Apl" here instead?